components/proxy/src/main/java/com/hotels/styx/infrastructure/configuration/json/ObjectMappers.java

```diff
@@ -0,0 +1,66 @@
+/*
+  Copyright (C) 2013-2018 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.infrastructure.configuration.json;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.hotels.styx.api.Id;
+import com.hotels.styx.api.client.Origin;
+import com.hotels.styx.api.client.OriginsSnapshot;
+import com.hotels.styx.api.service.BackendService;
+import com.hotels.styx.api.service.Certificate;
+import com.hotels.styx.api.service.ConnectionPoolSettings;
+import com.hotels.styx.api.service.HealthCheckConfig;
+import com.hotels.styx.api.service.RewriteConfig;
+import com.hotels.styx.api.service.StickySessionConfig;
+import com.hotels.styx.api.service.TlsSettings;
+import com.hotels.styx.infrastructure.configuration.json.mixins.BackendServiceMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.CertificateMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.ConnectionPoolSettingsMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.HealthCheckConfigMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.IdMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.OriginMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.OriginsSnapshotMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.RewriteConfigMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.StickySessionConfigMixin;
+import com.hotels.styx.infrastructure.configuration.json.mixins.TlsSettingsMixin;
+
+
+/**
+ * Collection of methods that helps to work with {@link ObjectMapper} objects in styx libraries.
+ */
+public class ObjectMappers {
```

Comment:
For an utility class like this, we would as per convention:

  1) Mark is a `final` to prevent subclassing, and
  2) Add a private constructor to prevent instantiation.